src/cljs/nr/cardbrowser.cljs

```diff
@@ -84,12 +129,8 @@
              (show-alt-art? true))
       (->> alt-arts
            (concat [""])
-           (map (fn [art] (if art
-                            (assoc card :art art)
-                            card)))
-           (map (fn [c] (if (:art c)
-                          (assoc c :display-name (str (:code c) "[" (alt-art-name (:art c)) "]"))
-                          c)))
+           (map #(if % (assoc card :art %) card))
+           (map #(if (not= "" (:art %)) (dissoc % :previous-versions) %))
```

Comment:
`(not= "" (:art %))` will be `true` if there's no `:art` key. Is that okay?

Reply:
Yes, I think that's okay. We have a list of known arts and I'm appending  "" to the end. I'm open to a smarter way to remove the key from all copies of the card except the "" one.

Reply:
I haven't spent enough time with this to know what is supposed to happen, lol, so I trust your judgement.